tests/store/tracking/test_file_store_logged_model.py

```diff
@@ -0,0 +1,605 @@
+        store.create_logged_model("123")
+    mock_experience = mock.Mock()
+    mock_experience.lifecycle_stage = LifecycleStage.DELETED
+    exp_id = store.create_experiment("test")
+    store.delete_experiment(exp_id)
+    with pytest.raises(
+        MlflowException,
+        match=rf"Could not create model under non-active experiment with ID {exp_id}",
+    ):
+        store.create_logged_model(exp_id)
+
+    with pytest.raises(MlflowException, match=r"A key name must be provided."):
+        store.create_logged_model(params=[LoggedModelParameter(None, "b")])
+
+    with pytest.raises(MlflowException, match=r"exceeds the maximum length"):
+        store.create_logged_model(params=[LoggedModelParameter("a" * 256, "b")])
+
+
+def test_set_logged_model_tags(store):
+    exp_id = store.create_experiment("test")
+    run_id = store.create_run(exp_id, "user", 0, [], "test_run").info.run_id
+    logged_model = store.create_logged_model(exp_id, "test_model", run_id)
+    assert logged_model.tags == {}
+    store.set_logged_model_tags(logged_model.model_id, [LoggedModelTag("tag_key", "tag_value")])
+    logged_model = store.get_logged_model(logged_model.model_id)
+    assert logged_model.tags == {"tag_key": "tag_value"}
+    store.set_logged_model_tags(logged_model.model_id, [LoggedModelTag("tag_key", "new_tag_value")])
+    logged_model = store.get_logged_model(logged_model.model_id)
+    assert logged_model.tags == {"tag_key": "new_tag_value"}
+    store.set_logged_model_tags(
+        logged_model.model_id, [LoggedModelTag("a", None), LoggedModelTag("b", 123)]
+    )
+    logged_model = store.get_logged_model(logged_model.model_id)
+    assert logged_model.tags == {"tag_key": "new_tag_value", "a": "", "b": "123"}
+
+
+def test_set_logged_model_tags_errors(store):
+    logged_model = store.create_logged_model()
+    with pytest.raises(MlflowException, match=r"Missing value for required parameter"):
+        store.set_logged_model_tags(logged_model.model_id, [LoggedModelTag(None, None)])
+    with pytest.raises(MlflowException, match=r"Names may only contain alphanumerics"):
+        store.set_logged_model_tags(logged_model.model_id, [LoggedModelTag("a!b", "c")])
+
+
+def test_get_logged_model(store):
+    experiment_id = store.create_experiment("test")
+    run_id = store.create_run(
+        experiment_id,
+        user_id="user",
+        start_time=0,
+        tags=[],
+        run_name="test_run",
+    ).info.run_id
+    logged_model = store.create_logged_model(
+        experiment_id=experiment_id,
+        name="test_model",
+        source_run_id=run_id,
+        tags=[LoggedModelTag("tag_key", "tag_value")],
+        params=[LoggedModelParameter("param_key", "param_value")],
+        model_type="dev",
+    )
+    fetched_model = store.get_logged_model(logged_model.model_id)
+    assert logged_model.model_uri == fetched_model.model_uri
+    assert logged_model.to_dictionary() == fetched_model.to_dictionary()
```

Comment:
it'd be nice if `LoggedModel` is `__eq__`able. We don't need to address this in this PR.